valhalla/app/controllers/concerns/valhalla/resource_controller.rb

```diff
@@ -67,7 +67,10 @@ def update
         change_set_persister.buffer_into_index do |persist|
           obj = persist.save(change_set: @change_set)
         end
-        after_update_success(obj, @change_set)
+        respond_to do |format|
```

Comment:
`after_update_success` already does the format negotiation.  I think it's better to have that happen in there instead of directly in the controller method.